coretex/_task/process_utils.py

```diff
@@ -0,0 +1,85 @@
+from typing import List, Any
+from pathlib import Path
+
+import logging
+import subprocess
+
+from threading import Thread
+
+from . import LoggerUploadWorker
+from ..entities import Log, LogSeverity
+
+
+runLogger = logging.getLogger("coretex-run")
+
+
+def _handleOutput(process: subprocess.Popen, worker: LoggerUploadWorker) -> None:
+    stdout = process.stdout
+    if stdout is None:
+        raise ValueError("stdout is None for subprocess")
+
+    while process.poll() is None:
+        line: str = stdout.readline().decode("UTF-8")
+        if line.strip() == "":
+            continue
+
+        worker.add(Log.create(line.rstrip(), LogSeverity.info))
+        runLogger.info(line.rstrip())
+
+
+def _handleError(process: subprocess.Popen, worker: LoggerUploadWorker, isEnabled: bool) -> None:
+    stderr = process.stderr
+    if stderr is None:
+        raise ValueError("stderr is None for subprocess")
+
+    lines: List[str] = []
+    while process.poll() is None:
+        line: str = stderr.readline().decode("UTF-8")
+        if line.strip() == "":
+            continue
+
+        worker.add(Log.create(line.rstrip(), LogSeverity.error))
+        lines.append(line.rstrip())
```

Comment:
This does not replicate Node behaviour. stderr logs get logged after the experiment has finished